ddtrace/internal/core.py

```diff
@@ -144,7 +144,6 @@ def addParent(self, context):
         if self.identifier == ROOT_CONTEXT_ID:
             raise ValueError("Cannot add parent to root context")
         self._parents.append(context)
-        self._data.update(context._data)
```

Comment:
Was this change intentional?

Reply:
Yes. It removes one of the ways that contribs can mutate the contents of an `ExecutionContext`.